src/components/survey/survey-reducer.ts

```diff
@@ -7,7 +7,39 @@ import {cioIdentify} from 'utils/cio-identify'
 const DEFAULT_FIRST_QUESTION = `biggest_path`
 const DEFAULT_FINAL_QUESTION = `thanks`
 
-export type SortingHatState = {
+export type Survey = {
+  [key: string]: SurveyQuestion | string
+}
+
+export type MultipleChoiceAnswer = {
+  answer: string
+  label: string
+  always_last?: boolean
+}
+
+export type SurveyQuestion = {
+  heading: string
+  subheading: string
+  type:
```

Comment:
I considered creating types for all of these... types, but we can talk more about that